ADALiOS/ADALiOS/ADInstanceDiscovery.m

```diff
@@ -164,16 +164,12 @@ -(void) validateAuthority: (NSString*) authority
         return;
     }
     
-    
-    dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_HIGH, 0), ^
-                   {
-                       //Nothing in the cache, ask the server:
-                       [self requestValidationOfAuthority:authority
-                                                     host:authorityHost
-                                         trustedAuthority:sTrustedAuthority
-                                            correlationId:correlationId
-                                          completionBlock:completionBlock];
-                   });
```

Comment:
Office needed this to happen on a background thread instead of the main thread. thats why we added dispatch_async.